server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -55,11 +55,16 @@ def deploy
       end
     end
 
+    def resource_name
+      'deployment'
```

Comment:
added this since Sat6/Foreman automatically check for "nested objects" if there is a `*_id` parameter, in our case `deployment_id`